subiquity/models/network.py

```diff
@@ -41,31 +42,56 @@ def render(self):
         # perfect solution because in principle there could be wired 802.1x
         # stuff that has secrets too but the subiquity UI does not support any
         # of that yet so this will do for now.
-        wifis = netplan['network'].pop('wifis', None)
-        r = {
-            'write_files': {
-                'etc_netplan_installer': {
-                    'path': 'etc/netplan/00-installer-config.yaml',
-                    'content': self.stringify_config(netplan),
+
+        # If host cloud-init version has no readable combined-cloud-config,
+        # default to False.
+        cloud_cfg = cloudinit.get_host_combined_cloud_config()
+        use_cloudinit_net = cloud_cfg.get('features', {}).get(
+            'NETPLAN_CONFIG_ROOT_READ_ONLY', False
+        )
+
+        if use_cloudinit_net:
+            r = {
+                'write_files': {
+                    'etc_netplan_installer': {
+                        'path': (
+                            'etc/cloud/cloud.cfg.d' '/90-installer-network.cfg'
```

Comment:
Why is there a space on this line?

Reply:
oops leftover from my accidental black formatting run due to a stray pre-commit hook setup I had. I manually reverted, but didn't squash the string all the way back to one from the multi-line that was setup due to earlier iteration of this change. Dropper/